Lib/multiprocessing/process.py

```diff
@@ -314,8 +314,10 @@ def _bootstrap(self):
         finally:
             threading._shutdown()
             util.info('process exiting with exitcode %d' % exitcode)
-            sys.stdout.flush()
-            sys.stderr.flush()
+            if sys.stdout is not None and not sys.stdout.closed:
```

Comment:
A minor issue, but I'd like to stick to the idiom already used in the previous PR: https://github.com/python/cpython/pull/4073/files#diff-d6a41af14d72f372f37ced0cc15c1f58L17

Reply:
Of course. It's the easier to ask for forgiveness than permission idiom.